precompiles/xcm/XCM_v2.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.8.0;
+
+/**
+ * @title XCM interface.
+ */
+interface XCM {
```

Comment:
What's the purpose of this interface if we have `xtokens`?

Reply:
This is the updated interface of the old XCM interface which supports `Multilocation` and `WeightV2` in addition to the new function `send_xcm`.
The main reason we are using it is to use the `send_xcm` which lets us send arbitrary xcm message which we can't do using the `xtokens`. 
Moreover, this will also be easier for consumers are already using the old interface to transition into this imo.

Reply:
There's no need to add this since you added `xtokens` precompile.
Instead of having one interface, with this update we'll have 3, where the 2nd one is an unncessary upgrade of the former `XCM_v1`.

If users want to use the new functionality, they should use the `xtokens`.

`send_xcm` can be added under a separate interface, or you can rename `xtokens` to `XCM_v2` instead and have it there. We could discuss this shortly tomorrow.

But I'm certain we don't want the `v2` interface in it's current form.

Reply:
Agree, it can be combined under a single interface for both xtokens and `send_xcm`